generators/setup-workspace/index.js

```diff
@@ -384,14 +389,28 @@ class Generator extends Base {
 
   end() {
     this.log('\n\nnext steps: ');
-    this.log(chalk.green('1. switch to the created directory: '), chalk.yellow(`cd ${this.cwd}`));
-    this.log(chalk.green('2. Open IDE (PyCharm or Visual Studio Code, Atom) and select:'), this.destinationPath(this.cwd));
+    this.log(chalk.green('- switch to the created directory: '), chalk.yellow(`cd ${this.cwd}`));
+    if (this.options.skip.includes('install')) {
+      this.log(chalk.green('- install npm dependencies: '), chalk.yellow('npm install'));
+    }
+    if (this.options.skip.includes('build')) {
+      this.log(chalk.green('- build docker containers: '), chalk.yellow('docker-compose build'));
+    }
+
+
+    this.log(chalk.green('- open IDE (PyCharm or Visual Studio Code, Atom) and select:'), this.destinationPath(this.cwd));
     this.log(chalk.green('   in case of Visual Studio Code, the following should also work: '), chalk.yellow('code .'));
-    this.log(chalk.green('3. start server docker container in the background (-d): '), chalk.yellow('docker-compose up -d'));
-    this.log(chalk.green('4. start client application in the foreground: '), chalk.yellow('npm start'));
-    this.log(chalk.green('5. open web browser and access: '), chalk.yellow('http://localhost:8080'));
+    this.log(chalk.green('- start server docker container in the background (-d): '), chalk.yellow('docker-compose up -d'));
+    this.log(chalk.green('- start client application in the foreground: '), chalk.yellow('npm start'));
+    this.log(chalk.green('- open web browser and access: '), chalk.yellow('http://localhost:8080'));
```

Comment:
I would keep the ordinal numbers since the user should perform them in this order. You can do this with a variable that you always increase after each log statement

```ts
    let stepCounter = 1;

    this.log('\n\nnext steps: ');

    this.log(chalk.green((stepCounter++) + '. Switch to the created directory: '), chalk.yellow(`cd ${this.cwd}`));

    if (this.options.skip.includes('install')) {
      this.log(chalk.green((stepCounter++) + '. install npm dependencies: '), chalk.yellow('npm install'));
    }
    if (this.options.skip.includes('build')) {
      this.log(chalk.green((stepCounter++) + '. build docker containers: '), chalk.yellow('docker-compose build'));
    }

    this.log(chalk.green((stepCounter++) + '. open IDE (PyCharm or Visual Studio Code, Atom) and select:'), this.destinationPath(this.cwd));

    // ...
```

Reply:
Furthermore, please use uppercase for the first letter of a sentence.